source/DonutModel.h

```diff
@@ -18,6 +18,14 @@ class DonutModel {
 	float angle;
 	/** Current turning thrust (stored to facilitate decay) */
 	float velocity;
+	/** Offset from bottom of ship when Jumping based on proportion of hallway */
+	float jumpOffset;
+	/** Whether donut is currently jumping */
+	bool jumping;
+	/** The ellapsed time since the beginning of the jump in seconds */
+	float timestamp;
```

Comment:
All the other member variables for jumping are named with the `jump` prefix; can we name this one like that too? `timestamp` isn't too descriptive and pollutes the namespace.